runtime.md

```diff
@@ -2,22 +2,26 @@
 
 ## State
 
-The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
-The runtime state is stored in a JSON encoded file.
-It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
-On Linux based systems the state information should be stored in `/run/opencontainer/containers`.
-The directory structure for a container is `/run/opencontainer/containers/<containerID>/state.json`.
-By providing a default location that container state is stored external applications can find all containers running on a system.
-
-* **`version`** (string) Version of the OCI specification used when creating the container.
-* **`id`** (string) ID is the container's ID.
-* **`pid`** (int) Pid is the ID of the main process within the container.
-* **`bundlePath`** (string) BundlePath is the path to the container's bundle directory.
-
+Metadata for containers are stored on disk so that external tools can consume and act on this information.
+It is recommended that this data be stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the metadata MUST be stored in `/run/opencontainer/containers`.
```

Comment:
On Fri, Oct 02, 2015 at 01:40:43PM -0700, Vincent Batts wrote:
> > +On Linux based systems the metadata MUST be stored in `/run/opencontainer/containers`.
> 
> Maybe we can say Linux/Unix? Thoughts?

“Systems with POSIX paths”?  I'm fine leaving this as “Linux” and
opting in other architectures as we add support for them.

Why don't we require a particular path on Windows?